services/app-users/src/user/user.controller.ts

```diff
@@ -50,4 +50,16 @@ export class UserController {
     console.log(`${this._name} handleBulkDelete Payload`, payload);
     return this.userService.handleBulkDelete(payload);
   }
+  @MessagePattern(USER_EVENTS.login)
```

Comment:
How does this work? Theres no login route in app-gateway. Shouldnt the whole authentication be its own service?

Reply:
So the app gateway will be handling all transactions between microservices?